src/index.html

```diff
@@ -7,8 +7,58 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:100,100italic,300,300italic,regular,italic,500,500italic,700,700italic,900,900italic" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <div class="header__wrap">
+          <a href="/">
+            <img
+              src="images/logo.png"
+              alt="Moyo_logo"
+              class="nav__logo">
+          </a>
```

Comment:
1. apply code styling [recommendations](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md) all over the page
2. this logo isn't an element of `nav` block
```suggestion
          <a href="/">
            <img
              src="images/logo.png"
              alt="Moyo_logo"
              class="header__logo"
            >
          </a>
```